src/transformers/modeling_utils.py

```diff
@@ -2386,14 +2386,12 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             modules_to_not_convert.extend(keep_in_fp32_modules)
 
             model = replace_8bit_linear(
-                model,
-                threshold=load_in_8bit_threshold,
-                modules_to_not_convert=modules_to_not_convert,
+                model, threshold=load_in_8bit_threshold, modules_to_not_convert=modules_to_not_convert
             )
 
-            model._is_int8_training_enabled = (
-                is_bitsandbytes_greater_0_37_0()
-            )  # training in 8-bit is only available in 0.37.0+
+            model._is_int8_training_enabled = version.parse(
+                importlib_metadata.version("bitsandbytes")
+            ) < version.parse("0.37.0")
```

Comment:
Here you can import the bitsandbytes to check the version

Reply:
Hmm the thing is that I think this is the only way to retrieve the version number from `bnb`: 
```
>>> bnb.__version__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'bitsandbytes' has no attribute '__version__'
```

Reply:
Oh, indeed looks like there is no maintained version in the init.